core/src/main/java/org/apache/gravitino/connector/PropertiesMetadata.java

```diff
@@ -102,10 +149,34 @@ default Object getOrDefault(Map<String, String> properties, String propertyName)
    * @return the default value object of the property.
    */
   default Object getDefaultValue(String propertyName) {
+    PropertyEntry<?> propertyEntry = getPropertyEntry(propertyName);
+
+    return propertyEntry.getDefaultValue();
+  }
+
+  /**
+   * Get the property entry of the property.
+   *
+   * @param propertyName The name of the property.
+   * @return the property entry object of the property.
+   * @throws IllegalArgumentException if the property is not defined.
+   */
+  default PropertyEntry<?> getPropertyEntry(String propertyName) throws IllegalArgumentException {
```

Comment:
Can you provide more comments on this method?  will we get the value of the longest key for prefix property and why?

Reply:
added